src/DetailsView/components/test-view-container.tsx

```diff
@@ -39,7 +39,7 @@ import { TestViewDeps } from './test-view';
 
 export type TestViewContainerDeps = {
     detailsViewActionMessageCreator: DetailsViewActionMessageCreator;
-    assessmentActionMessageCreator: AssessmentActionMessageCreator;
+    getAssessmentActionMessageCreator: () => AssessmentActionMessageCreator;
```

Comment:
nit: it seems like we're not using this here; we should probably remove it entirely.